fastapi/openapi/models.py

```diff
@@ -116,6 +116,8 @@ class SchemaBase(BaseModel):
     externalDocs: Optional[ExternalDocumentation] = None
     example: Optional[Any] = None
     deprecated: Optional[bool] = None
+    x_enum_varnames: Optional[List[str]] = Field(None, alias="x-enum-varnames")
+    x_enum_descriptions: Optional[List[str]] = Field(None, alias="x-enum-descriptions")
```

Comment:
Why was an alias needed?

Reply:
Because I need to export this information to OpenAPI. 
Currently I have in my enum the following code to generate the "x-enum-varnames" automatically, but without this variable/alias this information is ignored. 
The alias is required due to the variable name, just like the @Kludex commented.

My code:
```python
    @classmethod
    def __modify_schema__(cls, field_schema):
        field_schema.update({"x-enum-varnames": [item.name for item in cast(Iterable[Enum], enum)]})
```

Reply:
Ah yes I see now, the attribute is `x_enum_varnames` and the alias points to `x-enum-varnames` (with a dash, not an underscore).